tt_eager/tt_dnn/op_library/moreh_layernorm_backward/moreh_layernorm_backward_op.cpp

```diff
@@ -120,30 +133,34 @@ void MorehLayerNormBackwardGammaBetaGrad::validate(
 
 std::vector<Shape> MorehLayerNormBackwardGammaBetaGrad::compute_output_shapes(
     const std::vector<Tensor>& input_tensors) const {
-    // Inplace
+    // Not Implemented
     return {};
 }
 
 std::vector<Tensor> MorehLayerNormBackwardGammaBetaGrad::create_output_tensors(
-    const std::vector<Tensor>& input_tensors) const {
-    // Inplace
+    const std::vector<Tensor>& input_tensors, const std::vector<std::optional<Tensor>>& output_tensors) const {
+    if (output_tensors.at(0).has_value() && output_tensors.at(1).has_value()) {
+        log_debug(LogOp, "{}:{} use output tensor", __func__, __LINE__);
+        return {output_tensors.at(0).value(), output_tensors.at(1).value()};
+    }
+
+    TT_FATAL(false, "Create an optional tensor is not supported yet. fix this after the 9552 issue is addressed.");
```

Comment:
consider
```suggestion
    TT_FATAL(false, "Create output tensor is not supported yet. Fix this after the #9552 issue is addressed.");
```

Reply:
```C++
TT_FATAL(false, "Create output is not supported yet. Fix this after the #9552 issue is addressed.");
```
I have made the changes as you suggested.